### ISSUE TITLE
Raspberry Pi and Beef compiler

### ISSUE THREAD

**Issue by procfxgen** [beefytech/Beef#349]
Please make a raspberry version of the BeefBuild.exe tool.

**Comment by procfxgen** [comment]
It's possible to compile BeefBuild for pi4 ?

**Comment by bfiete** [comment]
The platform list is here https://www.beeflang.org/docs/platforms/

It seems like you'd want to cross-compile to the Rasberry Pi rather than compile directly on it, unless you had a particular need for that. The platform layer has not been ported to that platform yet, however.

**Comment by procfxgen** [comment]
There's BeefBuild that's already ported to work directly on pi ?

**Comment by bfiete** [comment]
No. Step 1 would be cross-compile support. Step 2 would be direct compile support.

**Comment by procfxgen** [comment]
I'm discovering "/bin/build.sh" for linux compilation, do you think I could adapt It and compile a raspberry version  ?
Do I need to compile LLVM or I could use apt package ?
Thanks.
